documentation/modules/ROOT/partials/modules/all-connectors/proc-verifying-the-connector-deployment.adoc

```diff
@@ -0,0 +1,183 @@
+If the connector starts correctly without errors, it creates a topic for each table that the connector is configured to capture.
+Downstream applications can subscribe to these topics to retrieve information events that occur in the source database.
+
+To verify that the connector is running, you perform the following operations from the OpenShift Container Platform web console, or through the OpenShift CLI tool (oc):
+
+* Verify the connector status.
+* Verify that the connector generates topics.
+* Verify that topics are populated with events for read operations ("op":"r") that the connector generates during the initial snapshot of each table.
+
+.Prerequisites
+
+* A {prodname} connector is deployed to {StreamsName} on OpenShift.
+* The OpenShift `oc` CLI client is installed.
+* You have access to the OpenShift Container Platform web console.
+
+.Procedure
+. Check the status of the `KafkaConnector` resource by using one of the following methods:
+* From the OpenShift Container Platform web console:
+.. Navigate to *Home -> Search*.
+.. On the *Search* page, click *Resources* to open the *Select Resource* box, and then type `*KafkaConnector*`.
+.. From the *KafkaConnectors* list, click the name of the connector that you want to check, for example *inventory-connector-{context}*.
+.. In the *Conditions* section, verify that the values in the *Type* and *Status* columns are set to *Ready* and *True*.
++
+* From a terminal window:
+.. Enter the following command:
++
+[source,shell,options="nowrap",subs="+attributes,quotes"]
+----
+oc describe KafkaConnector _<connector-name>_ -n _<project>_
+----
++
+For example,
++
+[source,shell,options="nowrap",subs="+attributes,quotes"]
+----
+oc describe KafkaConnector inventory-connector-{context} -n debezium
+----
++
+The command returns status information that is similar to the following output:
```

Comment:
I would strongly suggest we abbreviate the output. No need to print an entire resource here.  Alternatively I think this should also be possible

```oc get KafkaConnector inventory-connector-{context} -n debezium -o jsonpath='{.status.ConnectorStatus}'
```
Which should print just the Connector status section  **as JSON**

Reply:
> I would strongly suggest we abbreviate the output. No need to print an entire resource here. Alternatively I think this should also be possible
> 
> Which should print just the Connector status section **as JSON**

@jcechace 
The alternative that you suggest isn't showing in your comment. Let me know what you intended. Thanks.

Reply:
I abbreviated the output.